securitycenter/api/src/UpdateNotificationConfigSnippets.g.cs

```diff
@@ -33,10 +33,11 @@ public class UpdateNotificationConfigSnippets
         {
             NotificationConfigName = notificationConfigName,
             Description = "updated description",
-            PubsubTopicAsTopicName = pubsubTopic
+            PubsubTopicAsTopicName = pubsubTopic,
+            StreamingConfig = new NotificationConfig.Types.StreamingConfig { Filter = "state = \"INACTIVE\"" }
```

Comment:
```suggestion
            StreamingConfig = new StreamingConfig { Filter = "state = \"INACTIVE\"" }
```
And add the correspongin using if needed. We include the using directives in the sample so users will know what they need.

Reply:
fixed